collect_app/src/androidTest/java/org/odk/collect/android/support/StubOpenRosaServer.java

```diff
@@ -197,6 +204,10 @@ public List<File> getSubmissions() {
         return submittedForms;
     }
 
+    public int getAccesses() {
```

Comment:
We're not using this since I ended up replacing the UI test, but it feels like it could be useful in other tests. I reckon we keep it around?